tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -3037,7 +3013,7 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 					replicasLarge = 4
 					replicasSmall = 3
 				case v1alpha1.TiDBMemberType:
-					replicasLarge = 3
+					replicasLarge = 4
```

Comment:
Do we need `replicasLarge = 5` for `v1alpha1.TiKVMemberType` as above?

Reply:
seems ok